js/directives/day-block.js

```diff
@@ -0,0 +1,63 @@
+/* global angular */
+(function(){
+    'use strict';
+    var DayBlockDirective = function () {
+        return {
+            template : 
+            '<div class="day-bloc">'+
```

Comment:
yeah but as your directive does not use `replace: true`, you can write your scss with the `luid-day-block` element selector and remove the `.day-bloc` class entirely